.rubocop.yml

```diff
@@ -0,0 +1,67 @@
+AllCops:
```

Comment:
@tarcieri I see only that cellouloid enforces double-quote strings. The rubocop autogenerated config takes current state of the repo as a rule.
In DCell I've decided to have fine grained configuration. I would suggested the same thing in celluloid.
cc @digitalextremist